src/app/shared/components/autocomplete/autocomplete.component.html

```diff
@@ -0,0 +1,11 @@
+<input
+  class="govuk-input"
+  type="text"
+  name="{{id}}"
+  id="test-{{ id }}"
+  [attr.list]="'autocomplete-'+id"
```

Comment:
Can we think about something more meaningful as an id?
I guess the linter didn't complain about it but i was wondering if we could get something more unique from the data we receive perhaps instead?

Reply:
The id will be the name of the for control, for example: id="countryOfRegistration"